examples/ml-training/application.py

```diff
@@ -1,110 +0,0 @@
-import burr.core.application
```

Comment:
why remove this? this stub is fine -- just need to comment what needs to be added.